docs/UserGuide.md

```diff
@@ -320,60 +293,127 @@ Examples:
 
 ### Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons based on the given keywords.
+
+#### Format: 
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+* `find n/NAME [MORE NAMES]`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`.
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`.
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`.
+* `find a/ADDRESS [MORE ADDRESSES]` 
 
-![edit format](images/find0.png)
+* `find appt/APPOINTMENTDATE`
 
-* Precise expected outputs on success:
-* Successful find message. ‘X persons listed!’’, where X is the number of contacts listed.
-  The list of contacts whose names contain the input name is shown.
+#### Example commands:
+* `find n/John Alice`
+* `find a/Tokyo Geylang`
+* `find appt/2024-01-01`
 
-![edit format](images/find1.png)
+#### Acceptable values for each parameter:
+* Name: Valid string name
+* Address: Valid string address
+* Appointment date: Valid dates (after current date), in the format of `yyyy-mm-dd`
 
-* Precise expected outputs on failure:
-* If no name is input after the find command, an error message explaining the error will be shown,
-  reminding the user to follow the correct input format.
+#### _Name_
 
-![edit format](images/find2.png)
+![find_name format](images/find_n0.png)
 
-### Locating persons by address: `find_add`
+#### Precise expected outputs on success:
+* Message shown to the user: 
+```
+X persons listed!
+```
+where X is the number of contacts listed
+* The list of contacts whose names contain the input name is shown
 
-Finds persons whose address contain any of the given keywords.
+![find_name format](images/find_n1.png)
 
-Format: `find_add KEYWORD [MORE_KEYWORDS]`
+##### Precise expected outputs on failure:
+If no name is input after the find command, an error message explaining the error will be shown, 
+reminding the user to follow the correct input format.
```

Comment:
Small nits, should remove the full stop here for consistency! :D

Reply:
Fixed!